lms/djangoapps/instructor/views/api.py

```diff
@@ -1196,6 +1218,13 @@ def generate_registration_codes(request, course_id):
         internal_reference=internal_reference,
         customer_reference_number=customer_reference_number
     )
+    unit_price = (Decimal(sale_price)/course_code_number).quantize(Decimal('.01'), rounding=ROUND_DOWN)
```

Comment:
Hmm, I'm concerned about this approach.  Rounding down could make it so that the sum of all invoice items is less than the sale price.

I think it would make more sense to change the UI here so that sales admins enter the price *per* registration.  @chrisndodge does that make sense?